### DISCUSSION TITLE
v4 TelegramError export

### DISCUSSION THREAD

**Discussion by Autapomorph** [telegraf/telegraf#1289] [Ideas] [answered]
Do you mind exporting [TelegramError](https://github.com/telegraf/telegraf/blob/57f16c70901e3c73b9ab67933a926fedf7a2b11d/src/core/network/error.ts#L8)?

It's useful when we want to handle errors sent by Telegram apart from our code errors (smth like `error instanceof TelegramError`, like you do [here](https://github.com/telegraf/telegraf/blob/57f16c70901e3c73b9ab67933a926fedf7a2b11d/src/core/network/polling.ts#L45))

**Comment by wojpawlik** [comment]
Reasonable request. Just, the thought of exporting `core/` stuff makes me uneasy... but I guess the only alternative is `core/` depending on non-`core/`, which may lead to circular imports. Do you have anything to add @KnorpelSenf? I'll go ahead if you stay silent for 24 hours.

**Comment by wojpawlik** [comment, marked as answer]
Fixed, will be published in 4.0.2. For now `npm install telegraf/telegraf#a57dbcdd4e706681cdb390a086e1cf90a4cfee4d`.
